SPECS/podman/podman.spec

```diff
@@ -0,0 +1,3191 @@
+%global with_check 0
+%global with_debug 1
+%if 0%{?with_debug}
+%global _find_debuginfo_dwz_opts %{nil}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+%global provider github
+%global provider_tld com
+%global project containers
+%global repo %{name}
+# https://github.com/containers/%%{name}
+%global import_path %{provider}.%{provider_tld}/%{project}/%{repo}
+%global git0 https://%{import_path}
+# dnsname
+%global repo_plugins dnsname
+# https://github.com/containers/dnsname
+%global import_path_plugins %{provider}.%{provider_tld}/%{project}/%{repo_plugins}
+%global git_plugins https://%{import_path_plugins}
+%global commit_plugins 18822f9a4fb35d1349eb256f4cd2bfd372474d84
+%global shortcommit_plugins %(c=%{commit_plugins}; echo ${c:0:7})
+# gvproxy
+%global repo_gvproxy gvisor-tap-vsock
+# https://github.com/containers/gvisor-tap-vsock
+%global import_path_gvproxy %%{provider}.%{provider_tld}/%{project}/%{repo_gvproxy}
+%global git_gvproxy https://%{import_path_gvproxy}
+%global commit_gvproxy 4ee84d66bd86668f011733d8873989b5862bcd07
+%global shortcommit_gvproxy %(c=%{commit_gvproxy}; echo ${c:0:7})
+%global built_tag v4.1.1
+Summary:        Manage Pods, Containers and Container Images
+Name:           podman
```

Comment:
I've noticed this just now: please remove all specs, which are now duplicated in `SPECS-EXTENDED`. We build them in either core or extended, never in both.

Reply:
Will do!